resources/views/cruds/datagrids/_grid.blade.php

```diff
@@ -1,4 +1,7 @@
 @php
+    if ($model->child) {
```

Comment:
This adds a ton of requests to the page (3 requests per entity returned) which isn't great but we can't go around it since we need the child model when re-using the existing grid blade file.